CWE/decisions/Romania.txt

```diff
@@ -29,4 +30,195 @@ political_decisions = {
 			factor = 1
 		}
 	}
-}
\ No newline at end of file
+
+	oil_modernization_completion = {
+		picture = "rom_oil"
+		potential = {
+			AND = {
+				owns = 667
+				owns = 672
+				owns = 665
+			}
+			OR = {
+				has_country_flag = nicolae_ceausescu_flag
+				has_country_flag = rom_manescu
+				has_country_flag = gheorghe_apostol_romania
+				has_country_flag = rom_nicu
+				has_country_flag = rom_zoia
+			}
+			OR = { government = proletarian_dictatorship government = proletarian_dictatorship1 government = socialist_democracy } }
+			has_country_flag = romanian_oil_modernization_start
+			NOT = { has_country_flag = romanian_oil_modernization_complete }
+		}
+		
+		allow = {
+			NOT = { has_country_modifier = economic_malaise }
+			war = no
+			money = 100000
+		}
+		
+		effect = {
+			prestige = 100
+			667 = {
+				trade_goods = petroleum
+				any_pop = {
+					limit = { type = farmers }
+					pop_type = labourers
+				}
+				add_province_modifier = { name = oil_field duration = -1 } 			
```

Comment:
Effects from modernisations should not last indefinitely.

Reply:
I was copying from other events that do roughly what I want. Should it just use the standard 10 years then?

Reply:
Yes, 10 years should be good.